Dockerfile

```diff
@@ -47,4 +47,4 @@ RUN gem install bundler \
 # Install NodeJS packages using yarnpkg
```

Comment:
That is outdated then?

Reply:
Not really, yarn v3 uses as a base yarn classic (v1). The js huge code you have found is executed by yarn classic and so using `yarnpkg` (which is simlinked to yarn) will, at the same way, work